core/Model.php

```diff
@@ -15,4 +15,9 @@
     abstract public function getId();
 
     abstract public function setId($id): void;
+
+    public static function primitiveUUID() : int
+    {
+        return rand(10000000000, 99999999999);
```

Comment:
Throws following error: `rand() expects parameter 1 to be integer, float given`. But more important it is not unique in any way, so it can't be used as primary key generator. `rand()` can generate same number again.